core/nlp.go

```diff
@@ -45,7 +46,12 @@ func (c Classifier) ClassifyTokens(tokens []string) *nlp.StructuredInput {
 // context in the sentence.
 func buildClassifier() (Classifier, error) {
 	ner := Classifier{}
-	p := filepath.Join(os.Getenv("GOPATH"), "src", "github.com", "itsabot",
+	path := os.Getenv("GOPATH")
+	if path == "" {
```

Comment:
len(s)

Don't need to check for length more than once. The only length check should be in LoadEnvVars.  Every other time should assume that it's there.